.github/workflows/bump-version.yml

```
@@ -0,0 +1,54 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Bump Version
+
+# Controls when the action will run. 
+on:
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+    inputs:
+      Type:
+        dscription: 'Choose which version to bump'
+        required: true
+        default: 'patch'
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+
+      - name: Build
+        run: |
+          npm ci
+          npm run build
+          
+      - name: Bump Patch Version
+        run: |
+          git config --global user.email "version.bump@github.action.com"
+          git config --global user.name "asgardio-github-bot"
+          npm run bump-patch-version
+          git push --follow-tags
+        if: github.event.inputs.Type == 'patch'
+      
+      - name: Bump Minor Version
+        run: |
+          git config --global user.email "version.bump@github.action.com"
+          git config --global user.name "asgardio-github-bot"
+          npm run bump-minor-version
+          git push --follow-tags
+        if: github.event.inputs.Type == 'minor'
+        
+      - name: Bump Major Version
+        run: |
+          git config --global user.email "version.bump@github.action.com"
+          git config --global user.name "asgardio-github-bot"
+          npm run bump-major-version
```

Does actions allow this kind of syntax. So, we can get rid of the duplicated steps.
```suggestion
          npm run bump-${{ github.event.inputs.Type }}-version
```

---

Resolved by 025c819